lib/matplotlib/text.pyi

```diff
@@ -103,9 +103,9 @@ class Text(Artist):
     def get_usetex(self) -> bool: ...
     def set_parse_math(self, parse_math: bool) -> None: ...
     def get_parse_math(self) -> bool: ...
-    def set_fontname(self, fontname: str | Iterable[str]): ...
+    def set_fontname(self, fontname: str | Iterable[str]) -> None: ...
```

Comment:
Maybe should remove the `return ` from the implementation, and maybe call the canonically defined version of `set_fontfamily` as `set_family` is not type checkable as an alias.